.travis.yml

```diff
@@ -1,21 +1,14 @@
 language: php
 php:
     - 7.0
+    - 5.6
+    - 5.5
 
 env:
-    - LIBMEMCACHED_VERSION=1.0.18
-    - LIBMEMCACHED_VERSION=1.0.17
-    - LIBMEMCACHED_VERSION=1.0.16
-    - LIBMEMCACHED_VERSION=1.0.15
-    - LIBMEMCACHED_VERSION=1.0.14
-    - LIBMEMCACHED_VERSION=1.0.10
-    - LIBMEMCACHED_VERSION=1.0.8
-    - LIBMEMCACHED_VERSION=1.0.7
-    - LIBMEMCACHED_VERSION=1.0.6
-    - LIBMEMCACHED_VERSION=1.0.2
-    #- LIBMEMCACHED_VERSION=0.53
-    #- LIBMEMCACHED_VERSION=0.49
-    #- LIBMEMCACHED_VERSION=0.44
+    - LIBMEMCACHED_VERSION=1.0.18 # Debian Jessie / Ubuntu Xenial
+    - LIBMEMCACHED_VERSION=1.0.16 # RHEL / CentOS 7
+    - LIBMEMCACHED_VERSION=1.0.8  # Debian Wheezy / Ubuntu Trusty
+    - LIBMEMCACHED_VERSION=0.44   # Ubuntu Precise
```

Comment:
Is Precise still relevant for 3.0 (php7) code base? Unless it's meant to supported, and given tests fails with it, I wouldn't re-add it here.